data/sql_updates/create_linkage_names_view.sql

```diff
@@ -39,9 +39,23 @@ select
     l.link_date as link_date,
     l.load_date as load_date,
     l.expire_date as expire_date,
-    (select cand_nm from dimcandproperties where dimcandproperties.cand_sk = l.cand_sk order by candproperties_sk desc limit 1) as candidate_name,
-    (select cmte_nm from dimcmteproperties where dimcmteproperties.cmte_sk = l.cmte_sk order by cmteproperties_sk desc limit 1) as committee_name
-from dimlinkages l;
+    candprops.cand_nm as candidate_name,
+    cmteprops.cmte_nm as committee_name
+from dimlinkages l
+    left join (
+        select distinct on (cand_sk) cand_sk, cand_nm from dimcandproperties
+            order by cand_sk desc
```

Comment:
This and the clause below probably need the fix to add the properties key as the last ordering column, too.